cmd/mesh_bridge/main.go

```diff
@@ -0,0 +1,106 @@
+// +build !js
+
+// mesh_bridge is a short program that bridges two Mesh nodes. This is useful in cases where
```

Comment:
Can we rename this to `mesh-bridge` to match the name of our other executables (e.g. `mesh-bootstrap`, `peer-id-to-pub-key`)? This also seems to be the convention in the Go ecosystem (e.g. `golang-ci-lint`), though it isn't used by quite everyone.